marc_to_solr/lib/uri_ark.rb

```diff
@@ -3,6 +3,9 @@
 class URI::ARK < URI::Generic
   attr_reader :nmah, :naan, :name
 
+  ARK_REGEX = /\:\/\/(.+)\/ark\:\/(.+)\/(.+)\/?/.freeze
```

Comment:
Notice that I reverted to the original RegEx (more strict) version that we were using before PR 1559